pages/common/tar.md

```diff
@@ -4,34 +4,34 @@
 > Often combined with a compression method, such as gzip or bzip.
 > More information: <https://www.gnu.org/software/tar>.
 
-- Create an archive from files:
+- [C]reate an archive from [f]iles:
```

Comment:
Maybe we should use a lower case `C` because the options are case sensitive:
```suggestion
- [c]reate an archive from [f]iles:
```
Reference: https://github.com/tldr-pages/tldr-lint/pull/12#discussion_r481134812

Reply:
Done.